content/migrate/tf2pulumi.md

```diff
@@ -0,0 +1,401 @@
+        });
+        post.dataType = "json";
+    }
+    $.post(post)
+        .done(function(data) {
+            clearLanguageFiles(languageTextbox);
+
+            if (data.files) {
+                let filenames = Object.keys(data.files);
+                filenames.sort();
+                for (let i = 0; i < filenames.length; i++) {
+                    let fn = filenames[i];
+                    let code = data.files[fn];
+                    addLanguageFile(languageTextbox, fn, code);
+                }
+                $("#pulumi-code-download-button").show();
+            } else {
+                $("#couldnt-convert-code").show();
+            }
+
+            if (data.diagnostics) {
+                $("#pulumi-warnings").text(data.diagnostics);
+                $("#pulumi-warnings").show();
+            }
+        })
+        .fail(function(err) {
+            let errorText = "An unspecified error occurred";
+            if (err) {
+                if (err.responseText) {
+                    errorText = err.responseText;
+                    try {
+                        let errdata = JSON.parse(err.responseText);
+                        if (errdata.error) {
+                            errorText = errdata.error;
+                        }
+                    } catch {
+                        // ignore.
+                    }
+                }
+                errorText += " [" + err.statusText + " " + err.status + "]";
+            }
+            $("#pulumi-errors").text(errorText);
+            $("#pulumi-errors").show();
+            $("#couldnt-convert-code").show();
+        }).
+        always(function() {
+            $(document.body).css({"cursor": "default"});
+            $("#convert-button").css({"cursor": "pointer"});
+        });
+}
+
+// downloadCode downloads the currently converted code, if available.
+function downloadCode() {
+    let zip = new JSZip();
+    for (let fn of Object.keys(currentCodeFiles)) {
+        zip.file(fn, currentCodeFiles[fn]);
+    }
+    zip.generateAsync({ type: "blob" }).then(function(content) {
+        // Use FileSaver.js to save the file, triggering download in the user's browser.
+        saveAs(content, "tf2pulumi.zip");
+    });
+}
+
+window.onload = function() {
+    $(document).ready(function() {
+        // If there are querystring parameters populate the fields.
+        let tfUrl = getQueryVariable("url");
+        let tfCode = getQueryVariable("code");
+        if (tfUrl) {
+            $("#terraform-url").val(tfUrl);
+            setInputKind("url");
+        } else {
+            if (!tfCode) {
+                // Populate a simple default example.
+                let comment = "#"; // to suppress Markdown lint errors.
+                tfCode = `${comment} This Terraform sample provisions an AWS EC2 instance running Ubuntu.
+${comment} Choose a language and click CONVERT below -- or try replacing it with your own!
+
+provider "aws" {
+  region = "us-west-2"
+}
```

Comment:
Unfortunately this leads to a warning.  We can probably just leave it out (it's part of external configuration).